cpp/src/link_analysis/pagerank_impl.cuh

```diff
@@ -55,6 +55,13 @@ void pagerank(
   bool has_initial_guess,
   bool do_expensive_check)
 {
+  /**
+   * TODO: Add masking to the following prims:
+   * 1. count_if_v
+   * 2. count_if_e
+   * 3. transform_reduce_v
+   * 4.
+   */
```

Comment:
We have test code for `count_if_v` (https://github.com/rapidsai/cugraph/blob/branch-22.06/cpp/tests/prims/mg_count_if_v.cu).

Maybe for the first step, we may just update `count_if_v` to work with masks (this will require much less work than PageRank).
Then, we work on `count_if_e` before working on something more complex.